lightkurve/targetpixelfile.py

```diff
@@ -1077,6 +1077,28 @@ def _open_image(img, extension):
             factory.add_cadence(frameno=idx, flux=cutout.data, header=hdu.header)
         return factory.get_tpf(hdu0_keywords=allkeys, ext_info=ext_info, **kwargs)
 
+    def correct(self, method='pld', aperture_mask=None):
```

Comment:
I know we've had a discussion about this before, but we will now have
```
lc.correct()    # lightkurve.LightCurve
tpf.correct()  # lightkurve.LightCurve
```

@barentsen how do you feel about this? We could be more explicit and do something like this:
```
lc.SFFcorrect()    # lightkurve.LightCurve
tpf.PLDcorrect()  # lightkurve.LightCurve
```

And then calling e.g. `lc.PLDcorrect()` would raise an exception and direct you to use the TPF. I don't feel super strongly about this, but it might make it easier for people to understand.

Reply:
I retained the option to call `tpf.correct()`, but all tutorials and examples should use the new corrector API (#319). Let's chat about whether it's worth removing this function altogether.